eZ/Publish/Core/MVC/Symfony/Security/EventListener/SecurityListener.php

```diff
@@ -254,6 +253,7 @@ private function isMasterRequest( Request $request, $requestType )
      */
     protected function hasAccess( SiteAccess $siteAccess )
     {
-        return $this->securityContext->isGranted( new Attribute( 'user', 'login', array( 'valueObject' => $siteAccess ) ) );
+        // JV: Should this go true AuthorizationCheckerInterface?
+        return $this->repository->canUser( 'user', 'login', $siteAccess );
```

Comment:
@lolautruche As the service already dependent on repo, this was the easy way, however unsure if  AuthorizationChecker provides some session caching or something that I'm not seeing.